game.js

```diff
@@ -296,6 +335,42 @@ class Game extends Screen {
 		text(this.score, width / 2, LINE_SPACING)
 	}
 
+	drawTargetScore() {
+		textAlign(CENTER, CENTER)
+		textSize(30)
+		fill(255, 255, 250)
+		strokeWeight(4)
+		text('You need a total of ' + targetScore + ' points to win.', width / 2, LINE_SPACING * 13)
+		text('Only ' + (this.calculatePointsToGo()) + ' points to go!', width / 2, LINE_SPACING * 14)
+	}
+
+	drawProgressBar() {
+		push()
+		strokeWeight(40)
+		stroke(0, 255, 0, 100)
+		line(width / 2 - 100, LINE_SPACING + 750, width / 2 + 100, LINE_SPACING + 750)
+		stroke(0, 255, 0)
+		let percentComplete
+		let progressBarEndPoint = width / 2 - 100
+		if(this.score >= targetScore) {
+			percentComplete = 200
+		}
+		else {
+			percentComplete = Math.floor(this.score / targetScore * 200)
+		}
+		line(progressBarEndPoint, LINE_SPACING + 750, progressBarEndPoint + percentComplete, LINE_SPACING + 750)
+		pop()
+	}
+
+	calculatePointsToGo() {
+		if (this.score >= targetScore) {
```

Comment:
you can replace these six lines with
```suggestion
		return Math.max(0, targetScore - this.score)
```